cmd/rm-main.go

```diff
@@ -351,7 +344,7 @@ func mainRm(ctx *cli.Context) error {
 	// Support multiple targets.
 	for _, url := range ctx.Args() {
 		if isRecursive {
-			if isHostName(url) {
+			if !isAliasURLDir(url) && (strings.Count(url, string(filepath.Separator)) == 0) {
```

Comment:
same as above

Reply:
isAliasURLDir is used internally by the checkCopySyntax function. Here the intent is to make sure only the alias is entered, and it is at the root level not a subdirectory.